modules/ocs2/src/main/scala/gem/ocs2/Legacy.scala

```diff
@@ -0,0 +1,92 @@
+package gem.ocs2
+
+import gem.ocs2.pio.PioError._
+import gem.ocs2.pio.{PioError, PioParse}
+
+import scala.reflect.runtime.universe.TypeTag
+
+import scalaz._
+import Scalaz._
+
+
+/** Legacy system (telescope, instrument, observe, calibration) key and parser
+  * definitions.
+  */
+object Legacy {
```

Comment:
`Legacy` defines the mappings from strings that you find in OCS2 XML to their definitions in the new model.  I extracted it from where it was because it needs to be shared now by both the `SequenceDecoder` and the new `StaticDecoder`.  The static config is found in the OCS2 sequence's first step so it was convenient to just read it from the same place as the `SequenceDecoder`.